src/plugins/data/public/ui/dataset_selector/_dataset_table.scss

```diff
@@ -0,0 +1,11 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+.datasetTable {
```

Comment:
is this technically just apart of the explorer?

Reply:
yes. i can combine them